specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json

```diff
@@ -0,0 +1,842 @@
+            "type": "string"
+          },
+          "description": "List of reserved path prefixes"
+        }
+      },
+      "required": [
+        "prefix"
+      ],
+      "description": "Target for storage task assignment execution"
+    },
+    "ExecutionTrigger": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "RunOnce",
+            "OnSchedule"
+          ],
+          "x-ms-enum": {
+            "name": "TriggerType",
+            "modelAsString": false
+          },
+          "description": "The trigger type of the storage task assignment execution"
+        },
+        "parameters": {
+          "$ref": "#/definitions/TriggerParameters",
+          "description": "The trigger parameters of the storage task assignment execution"
+        }
+      },
+      "required": [
+        "type",
+        "parameters"
+      ],
+      "description": "Execution trigger for storage task assignment"
+    },
+    "TriggerParameters": {
+      "type": "object",
+      "properties": {
+        "startFrom": {
+          "type": "string",
+          "description": "When to start task execution. This is a required field when ExecutionTrigger.properties.type is 'OnSchedule'; this property should not be present when ExecutionTrigger.properties.type is 'RunOnce'"
+        },
+        "interval": {
+          "type": "integer",
+          "minimum": 1,
+          "format": "int32",
+          "description": "Run interval of task execution. This is a required field when ExecutionTrigger.properties.type is 'OnSchedule'; this property should not be present when ExecutionTrigger.properties.type is 'RunOnce'"
+        },
+        "intervalUnit": {
+          "type": "string",
+          "enum": [
+            "days"
+          ],
+          "x-ms-enum": {
+            "name": "IntervalUnit",
+            "modelAsString": false
+          },
+          "description": "Run interval unit of task execution. This is a required field when ExecutionTrigger.properties.type is 'OnSchedule'; this property should not be present when ExecutionTrigger.properties.type is 'RunOnce'"
+        },
+        "endBy": {
+          "type": "string",
+          "description": "When to end task execution. This is a required field when ExecutionTrigger.properties.type is 'OnSchedule'; this property should not be present when ExecutionTrigger.properties.type is 'RunOnce'"
+        },
+        "startOn": {
+          "type": "string",
+          "description": "When to start task execution. This is an optional field when ExecutionTrigger.properties.type is 'RunOnce'; this property should not be present when ExecutionTrigger.properties.type is 'OnSchedule'"
+        }
+      },
+      "description": "The trigger parameters update for the storage task assignment execution"
+    },
+    "ExecutionConstraints": {
+      "type": "object",
+      "properties": {
+        "abortOnFailures": {
```

Comment:
>abortOnFailures [](http://example.com/codeflow?start=9&length=15)

or 'abortOnFailureCount'? or 'abortAfterFailureCount'?

Reply:
Not required for this API version